ctakesclient/transformer.py

```diff
@@ -8,11 +21,11 @@
 from ctakesclient.exceptions import ClientError
 from ctakesclient.typesystem import Polarity
 
-###############################################################################
-#
-# https://github.com/Machine-Learning-for-Medical-Language/cnlp_transformers#negation-api
-#
-###############################################################################
+
+class TransformerModel(enum.Enum):
+    # Use the cnlpt model slug as a value, in case that's convenient for any consumers
+    NEGATION = "negation"
+    TERM_EXISTS = "termexists"
```

Comment:
with my best practices hat on, i would probably ask the name of the downstream model to change for PEP-8 reasons. i would defer to Tim, but i don't think this is a specific term of art. But i'm fine with this separation here in the enum for the time being.

Reply:
Yeah maybe "term of art" was a bad choice of words. I meant not like, in the ML space at large, but rather because downstream called the model that, does it become a term of art for us in using it.

I don't know as if it's a PEP-8 issue downstream - that's more concerned with Python variables and such. But downstream does use the word without spaces in a few non-code places. The hugging face model name (`termexists_pubmedbert_ssm`), the REST endpoint (`/termexists/process`), the Docker image name (`termexists-latest-cpu` -- which I realize is my own doing but in that case it felt appropriate to follow the code), and internally in the code.

But here, it felt like a reasonable place to abstract away the lower-level name and use a more human friendly name (i.e. a single space). But maybe we are already thinking about this all too hard.

Reply:
our good old friend Thinking About This Too Hard! yeah, this seperation point is fine